src/components/AccountSettings.tsx

```diff
@@ -0,0 +1,190 @@
+            //     console.log(updatedProfile)
+            //     console.log(userProfile)
+            // }
+            cancelEdit()
+        } catch(err:any) {
+            displayToast(err.reason, 'danger')
+        }
+    }
+
+
+    useEffect(() => {
+        const getData = async() => {
+            try {
+                const mask =
+                  SynapseConstants.USER_BUNDLE_MASK_ORCID |
+                  SynapseConstants.USER_BUNDLE_MASK_USER_PROFILE |
+                  SynapseConstants.USER_BUNDLE_MASK_IS_VERIFIED |
+                  SynapseConstants.USER_BUNDLE_MASK_VERIFICATION_SUBMISSION
+
+                const bundle:UserBundle = await getMyUserBundle(
+                    mask,
+                    accessToken
+                )
+
+                setUserProfile(bundle.userProfile)
+                setOrcid(bundle.ORCID)
+                setVerfied(bundle.isVerified)
+                console.log(bundle.verificationSubmission)
+            } catch (err: any) {
+                displayToast(err.reason as string, 'danger')
+            }
+        }
+        getData()
+        console.log('hello')
+    }, [accessToken])
+
+    // Closes any forms and resets the fields.
+    const cancelEdit = () => {
+        setEditUsername(false)
+        setUpdatedUsername('')
+        setEditEmail(false)
+        setUpdatedEmail('')
+        setChangePW(false)
+    }
+
+    // Component to edit fields (username / email)
+    const EditField = (label: string, updatedValue: string, updateFn: Function) => {
+        return(
+        <div>
+            <FormGroup className='required'>
+                <FormLabel>{label}</FormLabel>
+                <FormControl 
+                    onChange={e => updateFn(e.target.value)} 
+                    value = {updatedValue}/>
+                <Button style={{width:'190px'}} className='btn-container emptyButton' onClick={cancelEdit}>Cancel</Button>&nbsp;&nbsp;
+                <Button style={{width:'190px'}} className='btn-container' variant='secondary' onClick={onUpdateUserProfile}>Save Changes</Button>
+            </FormGroup>
+        </div>
+        )
+    }
+
+    return(  
+        <div className="bootstrap-4-backport blue-background AccountSettings">
+            <Container>
+                <Row>
+                    <Col sm={3}>
+                        {verified ? 
+                        <div>
+                            <img src={VerifedAccount} alt="verified"/> 
+                            <p style={{marginTop:'8px'}}>Verified Account</p>
+                        </div>
+                        : 
+                        <div>
+                            <img src={StarterAccount} alt="starter"/>
+                            <p style={{marginTop:'8px'}}>Starter Account</p>
+                        </div>
+                        }
+                    </Col>
+                    <Col sm={9}>
+                        <table style={{display:"inline-block"}}>
```

Comment:
Haven't seen a table used for layout purposes in a long time!  Recommend changing this to a css grid (since this is not tabular data).